src/Discord.Net.Interactions/Utilities/ApplicationCommandRestUtil.cs

```diff
@@ -223,7 +223,7 @@ public static ApplicationCommandProperties ToApplicationCommandProps(this IAppli
                     Name = command.Name,
                     Description = command.Description,
                     IsDefaultPermission = command.IsDefaultPermission,
-                    DefaultMemberPermissions = (GuildPermission)command.DefaultMemberPermissions.RawValue,
+                    DefaultMemberPermissions = command.DefaultMemberPermissions.RawValue == 0 ? new Optional<GuildPermission>() : (GuildPermission)command.DefaultMemberPermissions.RawValue,
```

Comment:
Is it possible to use the approach in L108 here as well. It would be nice if we can keep everything consistent

Reply:
> Is it possible to use the approach in L108 here as well. It would be nice if we can keep everything consistent

L108's Builder uses `GuildPermission?`, while these XCommandProperties.DefaultUserPermissions uses `Optional<GuildPermission>`. The Sanitize helper method is geared towards the former, not the latter. It's already a bit inconsistent (the Builder on L108 ends up converting to Optional<> in the constructor anyway). I can definitely make a secondary method for this data type however - let me know.

Reply:
@Cenngo any additional feedback here, or leave as-is ?